backend/src/routes/feedback/feedback.messages.post.js

```diff
@@ -1,3 +1,19 @@
+import { formatDate } from '../utils';
+
+const getDetailedMessage = (feedback, { text }) => {
+  const details =
```

Comment:
Use string interpolation instead of concatenation ie 
const details = `Ticket id: ${feedback.id}`...